ftl/node/benchmark/main.py

```diff
@@ -86,11 +97,12 @@ def main():
         if not os.path.isfile(node_builder_path):
             node_builder_path = 'bazel-bin/ftl/node_builder.par'
 
-        cmd = subprocess.Popen([node_builder_path,
-                                '--base', args.base,
-                                '--name', args.name,
-                                '--directory', args.directory,
-                                '--no-cache'], stderr=subprocess.PIPE)
+        cmd = subprocess.Popen(
```

Comment:
You might be able to simplify this a bit with subprocess.check_output.

Reply:
done